content/en/internal/collaboration-tools/sso/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "EGI SSO"
+type: docs
+weight: 20
+description: "EGI SSO usage"
+---
+
+[EGI SSO](https://sso.egi.eu) is a legacy central Identity Provider mainly
+intended to access the [EGI Collaboration Tools](../../collaboration-tools).
+
+> For most services people should look into using federated authentication via
+> [EGI Check-in](../../../users/aai/check-in).
+
+> Still, for some services not yet integrated with Check-in, an EGI SSO account
+> should be used, in that case you can
+> [create an EGI SSO account](https://sso.egi.eu/admin/email).
+
+## Features
+
+- Central Identity Provider allowing to access EGI Collaboration Tools
+- SSO group management
```

Comment:
is "SSO" here in this line bringing any actual meaning?

Reply:
Right I will just leave group management.